rules/do-you-use-a-data-warehouse/rule.md

```diff
@@ -0,0 +1,76 @@
+
+## Example
+The business *Northwind Chips & Cream* sells hot chips and ice cream from their fleet of food trucks, operating across Australia.
+
+The company wants to analyze its sales data across geographic locations and weather factors. They want to optimize their food truck distribution according to the highest grossing suburbs in each city for each day's conditions.
+
+* The company's sales data is stored in an Accounting system (ACME).
+* The trucks' locations are stored in a Fleet Management Library (FML).
+* The weather data comes from the country's weather API (BOM).
+
+A Data Warehouse is a compelling recommendation, because:
+
+* Analytical goals are clearly defined
+* Time variant data is used
+* Daily forecasts incentivize automation
+* Disparate data sources require aggregation
+* No real-time data ingestion required
+
+
+## How do I create my own Data Warehouse?
+
+### Define Business Goals and Objectives
+The first step is to clearly define the business goals and objectives that the Data Warehouse will support. This involves understanding the specific analytical and reporting requirements of the organization. Identify the key areas where data insights can drive value, such as improving operational efficiency, enhancing customer experience, or enabling strategic decision-making. By aligning the Data Warehouse project with the business goals, you ensure that it addresses the specific needs of the organization.
+
+### Conduct Data Assessment
+Conducting a thorough data assessment is crucial to understand the current state of data within the organization. This step involves identifying the sources of data, data quality issues, data formats, data volume, and data integration requirements. Assessing the existing data infrastructure helps identify any gaps or inconsistencies that need to be addressed before implementing a Data Warehouse. It also helps in understanding the scope and complexity of data integration efforts required for successful implementation.
+
+### Develop a Data Warehouse Strategy and Architecture
+After defining business goals and assessing data, the next step is to develop a comprehensive Data Warehouse strategy and architecture. This involves determining the appropriate architecture for the Data Warehouse, including decisions related to data modeling, data extraction, transformation, loading processes, and data storage. Consider factors such as scalability, performance, security, and future growth requirements. Additionally, establish a governance framework for data management, including data ownership, access controls, and data privacy considerations. Developing a clear strategy and architecture provides a roadmap for the implementation of the Data Warehouse and ensures its alignment with the organization's long-term goals.
+
+## Popular Data Warehouse products
```

Comment:
Should we add SQL Server here too?

It's not uncommon to have warehouses built in regular SQL Server databases, probably less common but also still used is SQL Server Analysis Services, which should probably also be in the list for on-premises customers

Reply:
2005 called. It wants its architecture back. :)